changelog.d/7761.feature

```diff
@@ -0,0 +1 @@
+Added ring for group calls
```

Comment:
We don't need this file for downstream changes

Reply:
Removed this file